docs/configuration.asciidoc

```diff
@@ -79,6 +80,26 @@ NOTE: The service name must conform to this regular expression: ^[a-zA-Z0-9 _-]+
 | Name of the entry assembly       | String 
 |============
 
+[float]
```

Comment:
Nice, thanks for adding the docu! LGTM, @bmorelli25 Could you please also look at this?